### ISSUE TITLE
Cannot load browser, or any file

### ISSUE THREAD

**Issue by blank-manash** [emacs-eaf/emacs-application-framework#983]
In short, eaf is not working. The demo page sometimes works, and sometimes doesn't. But the main framework like the browser doesn't load at all. This is the error that is shown in the messages buffer

```
[EAF] Buffer id '1d95-ea82-5ab5-62e7-88b3-e11a-9df2' not exists, rebuild EAF buffer. [2 times]
```
I tested this both on my configured emacs and ```emacs -Q``` with minimal `load-path` and `(require 'eaq) (require 'eaq-browser)` both show the same error

 - Distro and DE/WM: Ubuntu 20.04 with i3wm
 - Versions of Dependencies: Using PyQt6
 - M-x emacs-version: 28.1

The eaf buffer is very long, here are the last output

```
Traceback (most recent call last):
  File "/home/manash/.emacs.d/site-lisp/emacs-application-framework/core/utils.py", line 50, in on_signal_received
    self._func(obj, *args, **kwargs)
  File "/home/manash/.emacs.d/site-lisp/emacs-application-framework/eaf.py", line 153, in new_buffer
    self.create_buffer(buffer_id, url, module_path, arguments)
  File "/home/manash/.emacs.d/site-lisp/emacs-application-framework/eaf.py", line 171, in create_buffer
    app_buffer = module.AppBuffer(buffer_id, url, arguments)
  File "/home/manash/.emacs.d/site-lisp/emacs-application-framework/app/browser/buffer.py", line 79, in __init__
    self.buffer_widget.init_dark_mode_js(__file__, get_emacs_var("eaf-browser-text-selection-color"))
  File "/home/manash/.emacs.d/site-lisp/emacs-application-framework/core/webengine.py", line 703, in init_dark_mode_js
    js_string = open(os.path.join(os.path.dirname(module_path), "node_modules", "darkreader", "darkreader.js")).read()
FileNotFoundError: [Errno 2] No such file or directory: '/home/manash/.emacs.d/site-lisp/emacs-application-framework/app/browser/node_modules/darkreader/darkreader.js'
```

These are some initial parameters

```
qt.webenginecontext: 

GL Type: desktop
Surface Type: OpenGL
Surface Profile: CompatibilityProfile
Surface Version: 4.6
QSG RHI Backend: OpenGL
Using Supported QSG Backend: yes
Using Software Dynamic GL: no
Using Multithreaded OpenGL: yes

Init Parameters:
  *  application-name eaf.py 
  *  browser-subprocess-path /home/manash/.local/lib/python3.8/site-packages/PyQt6/Qt6/libexec/QtWebEngineProcess 
  *  create-default-gl-context  
  *  disable-features ConsolidatedMovementXY,InstalledApp,BackgroundFetch,WebOTP,WebPayments,WebUSB,PictureInPicture 
  *  disable-setuid-sandbox  
  *  disable-speech-api  
  *  disable-web-security  
  *  enable-features NetworkServiceInProcess,TracingServiceInProcess 
  *  enable-gpu-rasterization  
  *  enable-native-gpu-memory-buffers  
  *  enable-threaded-compositing  
  *  ignore-gpu-blocklist  
  *  in-process-gpu  
  *  use-gl desktop 
```

**Comment by blank-manash** [comment]
The buffer id is completely random at times.

**Comment by manateelazycat** [comment]
https://github.com/emacs-eaf/emacs-application-framework#report-bug

read above doc.

**Comment by manateelazycat** [comment]
did you really read content of README?

why not use install-eaf.py install EAF?

**Comment by MatthewZMD** [comment]
> did you really read content of README?
> 
> why not use install-eaf.py install EAF?

I think it's likely that his initial `./install-eaf.py` failed at the `npm install` step but went unnoticed.

**Comment by blank-manash** [comment]
> did you really read content of README?
> 
> why not use install-eaf.py install EAF?

Hi I did that step of course and I have read the README
I think `npm install` did not work correctly by the time was it run because why would it otherwise not install only one package but install the rest of them.

**Comment by workcomplete** [comment]
I had the same issue solved by the below, using emacs 28.1 (doomemacs) on macos.

> I went to `~/.emacs.d/site-lisp/emacs-application-framework/app/browser/` and did a `npm install darkreader` which enabled the browser to work correctly, But I'm still having issues with the terminal working. This is the output of **eaf** buffer
> 
> ```
> Uncaught ReferenceError: Terminal is not defined
> [New Thread 0x7fff065ae700 (LWP 1481651)]
> [Thread 0x7fff065ae700 (LWP 1481651) exited]
> ```
